reverse-proxy.md

```diff
@@ -271,6 +271,7 @@ server {
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_read_timeout 3600; # set it to the same value as this env: https://github.com/nextcloud/all-in-one#how-to-adjust-the-max-execution-time-for-nextcloud
         client_max_body_size 0;
```

Comment:
I personally would leave on zero by default (so no change on this)
```suggestion
        client_max_body_size 10G; # set it to the same value as this env: https://github.com/nextcloud/all-in-one#how-to-adjust-the-upload-limit-for-nextcloud (or set it to 0 to allow any size)
```

Reply:
So then maybe just also set proxy_read_timeout to 0 since we already have both limits in apache/php

Reply:
> So then maybe just also set proxy_read_timeout to 0 since we already have both limits in apache/php

we could set it to 1d (one Day) by default in this config

Reply:
hm yeah, then maybe 2d which is likely close to unlimited or rather more?

Reply:
I would say one day is more than enough, I would say no one would upload a file which is so big that the upload takes over one day